blueocean-core-js/src/js/config.js

```diff
@@ -35,6 +36,10 @@ export default {
         return config.jenkinsConfig || {};
     },
 
+    getOrganization() {
+        return encodeURIComponent(organization.name);
```

Comment:
Would there be any value to having separate methods for the organization and organizationName? We might also want to consider whether the organization name should always be encoded, in case we'd like to display it in the UI.